src/test/java/bb/example/PersonDatabaseTest.java

```diff
@@ -1,42 +1,44 @@
 package bb.example;
 
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.mockito.InjectMocks;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
 
+@ExtendWith(MockitoExtension.class)
```

Comment:
You need to add the mockito extension in order to have mocks created using @Mock annotation

Reply:
isnt method MockitoAnnotations.openMocks(this) initializing mock annotation equivalently as the MockitoExtension?